code/modules/mob/living/simple_animal/hostile/headcrab.dm

```diff
@@ -28,6 +30,13 @@
 			if(H.stat == DEAD || (!H.check_death_method() && H.health <= HEALTH_THRESHOLD_DEAD))
 				Zombify(H)
 				break
+	for(var/mob/living/simple_animal/K in oview(src, 1)) //Only for corpse right next to/on same tile
```

Comment:
This is going to do a scan of its vicinity every Life() tick.

Reply:
I honestly have no better solution to this currently.